contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue

```diff
@@ -173,11 +165,7 @@
       });
       const Convertor = tmpEditor.convertor.constructor;
       class CustomConvertor extends Convertor {
-        toMarkdown(html, toMarkOptions) {
-          // Delete the img rule to prevent images from getting
-          // pre-parsed (need to factor in width and height)
-          delete toMarkOptions.renderer.rules.IMG;
-          let content = super.toMarkdown(html, toMarkOptions);
```

Comment:
Since we're not relying on raw `img` tags anymore, I don't think this should be needed anymore. It was also causing the markdown in `MarkdownImageField` components to get escaped, adding lots of extra "\" characters.

Reply:
It makes sense to me